docs/README_RECON.md

```diff
@@ -0,0 +1,263 @@
+| jdbc_reader_options | string                | jdbc_reader_option,that helps to parallelize the data read from jdbc sources based on the given configuration.For more info [jdbc_reader_options](#jdbc_reader_options)                                                          | optional(default=None) | "jdbc_reader_options": {"number_partitions": 10,"partition_column": "s_suppkey","upper_bound": "10000000","lower_bound": "10"} |
+| select_columns      | list[string]          | list of columns to be considered for the reconciliation process                                                                                                                                                                  | optional(default=None) | ["id","name","address"]                                                                                                        |
+| drop_columns        | list[string]          | list of columns to be eliminated from the reconciliation process                                                                                                                                                                 | optional(default=None) | ["comment"]                                                                                                                    |
+| column_mapping      | list[ColumnMapping]   | list of column_mapping,that helps in resolving column name mismatch b/w src and tgt eg: "id" in src  and "emp_id" in tgt.For more info [column_mapping](#column_mapping)                                                         | optional(default=None) | "column_mapping": [{"source_name": "id","target_name": "emp_id"}]                                                              |
+| transformations     | list[Transformations] | list of user-defined transformations ,that can be applied on src and tgt columns in case of any incompatibility data types or explicit transformation applied during migration.For more info [transformations](#transformations) | optional(default=None) | "transformations": [{"column_name": "s_address","source": "trim(s_address)","target": "trim(s_address)"}]                      |
+| thresholds          | list[Thresholds]      | list of threshold conditions, that can be applied on tgt columns to match the minor exceptions in data.It supports percentile,absolute, and date fields.For more info [thresholds](#thresholds)                                  | optional(default=None) | "thresholds": [{"column_name": "sal","lower_bound": "-5%","upper_bound": "5%","type": "integer"}]                              |
```

Comment:
Actually there is no `integer` in sqlglot datatypes .. so change it to `int`

Reply:
done